src/components/UploadZzal/ImageUpload.tsx

```diff
@@ -0,0 +1,115 @@
+import { Fragment, useRef, useState, DragEvent, ChangeEvent } from "react";
+import { Upload } from "lucide-react";
+import { cn } from "@/utils/tailwind";
+import ZzalCard from "@/components/common/ZzalCard";
+
+type DivDragEvent = DragEvent<HTMLDivElement>;
```

Comment:
저희 컨벤션으로 interface를 주로 사용하기로 했었는데,
다음과 같이 인터페이스로 작성하는 것이 좋을까요?
```suggestion
interface DivDragEvent extends DragEvent<HTMLDivElement>
```

Reply:
음 따로 확장해서 사용하지는 않는거 같아서 애매하긴 하네요? 저는 사실 `DragEvent<HTMLDivElement>`이런건 따로 할당하지 않고 있는 그대로 쓰는 편이에요! 있는 그대로 사용한다면 react에서 가져온 돔 타입이구나 하는데, 새로운 이름으로 할당하면 react에서 가져온 타입이라는 생각이 안들어서 한 번 더 확인해봐야 해서요